test/test_private_offer_stage.js

```diff
@@ -0,0 +1,76 @@
+var Token = artifacts.require("MFC_Token");
+var Controller = artifacts.require("ICO_controller");
+var WhitelistedCrowdsale = artifacts.require("WhitelistedCrowdsale");
+var BigNumber = require('bignumber.js');
+var wait = require('./utils').wait;
+var Web3 = require('web3');
+var web3 = new Web3(new Web3.providers.HttpProvider("http://localhost:8545"));
+
+contract('Private Offer', async function (accounts) {
+
+
+    it("test Private Offer", async function () {
+
+        let controller_instance = await Controller.deployed();
+        let startTime = Math.ceil(Date.now() / 1000);
+        let endTime = Math.ceil(Date.now() / 1000) + 100;
+        let initialBalance = web3.eth.getBalance(accounts[5]);
+        let escrowAddress = accounts[5];
+        let buyerAddress = accounts[2];
+        await controller_instance.startPrivateOffer(
+            startTime,
+            endTime,
+            escrowAddress);
+        let privateOffer = new WhitelistedCrowdsale(await controller_instance.privateOffer.call());
+        let token = new Token(await controller_instance.token.call());
+        wait(5);
```

Comment:
I saw we were using `wait(2)` in another test, can we have `2` here as well? The delay should be as small as possible.